routers/api/v2/users.go

```diff
@@ -0,0 +1,92 @@
+package v2
+
+import (
+	"github.com/gin-gonic/gin"
+	"github.com/pkg/errors"
+	"github.com/unicsmcr/hs_auth/routers/api/models"
+	"github.com/unicsmcr/hs_auth/services"
+	"go.uber.org/zap"
+	"net/http"
+)
+
+// POST: /api/v2/users/login
+// x-www-form-urlencoded
+// Request:  email string
+//           password string
+// Response: token string
+// Headers:  Authorization <- token
+func (r *apiV2Router) Login(ctx *gin.Context) {
+	email := ctx.PostForm("email")
+	if email == "" {
```

Comment:
Nit: This uses `== ""` for the string comparison, where as the `Register` function uses the `len()` function.